include/spdlog/details/registry.h

```diff
@@ -131,15 +134,23 @@ template <class Mutex> class registry_t
         _err_handler = handler;
     }
 
```

Comment:
worker_warmup_cb and worker_teardown_cb are now shared by ALL the threads in the pool.
I wonder if this could be a problem for some users or not.